manifests/h/HydrologicEnginerringCenter/HEC-RAS/6.1.0.0/HydrologicEnginerringCenter.HEC-RAS.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: HydrologicEnginerringCenter.HEC-RAS
+PackageVersion: 6.1.0.0
+InstallerSwitches:
+  Silent: /S /v/qn
+Installers:
+- Architecture: x86

```

I'm not sure why it's not installing silently. Maybe try using x64 architecture?
```suggestion
- Architecture: x64
```